### DISCUSSION TITLE
any update

### DISCUSSION THREAD

**Discussion by sharafdin** [dugsiiyeinc/dhaqsoPay#25] [General]
Dhalinyaro xagee ayuu wax idin maraa ?

Qolada `server`-ka ma bilaabeen howshooda ?

Qolada `extension`-kana xagee marinaayaan ?

@dugsiiyeinc/dhaqsopay-extension-developers 

@dugsiiyeinc/dhaqsopay-server-developers 

@dugsiiyeinc/dhaqsopay-website-developers

**Comment by shiinedev** [comment]
@sharafdin  aniga hadda waxan ku so daray menu pull requst ayan soo diray ee aqbal bro

**Reply by sharafdin** [reply]
review ayaan kuso siyay akhii halmar iga soo xaqiiji

**Reply by shiinedev** [reply]
war waba hilmame  hada ayan badalaya ee isug

**Reply by sharafdin** [reply]
> war waba hilmame hada ayan badalaya ee isug

si fudud lee base branch u bdl pull request dhan ha deleted gareen Insha'Allah!

**Reply by shiinedev** [reply]
wan u badalay inkasto conflict uu jiro

**Reply by sharafdin** [reply]
is xaliya Insha'Allah!

**Comment by abdinasir-Tman** [comment]
@sharafdin  bro waan howl galnay structure ka ayaan dejinay apis ka ayaa inoo harsan

**Reply by sharafdin** [reply]
alright Thanks

**Comment by A-M-Ahmed** [comment]
Waa wadnaa eng waxyar noo hary insha allah weekgn wakdhmnena xageena

**Reply by sharafdin** [reply]
akhii iskudeya inaad oga dhamaataan berito si aan comments idin dhaafno Insha'Allah!

**Reply by A-M-Ahmed** [reply]
2 noga dhig Saturday wan kdhmneynaa insha allah

**Reply by sharafdin** [reply]
haye akhii Insha'Allah!
